prow/new-e2e-rbac_no_auth.sh

```diff
@@ -21,8 +21,6 @@
 #                                     #
 #######################################
 
-# Exit immediately for non zero status
```

Comment:
don't remove that otherwise you have to check every command that you run. Instead consider the following:
``` TEST="$(return 1 || echo 'FAILED')"
 12 
 13 [[ ${TEST} == 'FAILED' ]] && echo "Command failed"
```